# HG changeset patch
# User Rik <rik@octave.org>
# Date 1354293853 28800
#      Fri Nov 30 08:44:13 2012 -0800
# Node ID a3189d32990696edd80cdab912682bff670b557a
# Parent  c2b381deca6b01246a73cd55ac93ad9b51961325
Update error_ids.m and add it to the build system.

* error_ids.m: Update docstring.

* scripts/miscellaneous/module.mk: Add error_ids.m to list of miscellaneous
functions.

diff --git a/scripts/miscellaneous/error_ids.m b/scripts/miscellaneous/error_ids.m
--- a/scripts/miscellaneous/error_ids.m
+++ b/scripts/miscellaneous/error_ids.m
@@ -16,32 +16,33 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @cindex error ids
 ##
 ## @table @code
 ## @item Octave:invalid-context
-## Idicates the error was generated by an operation that cannot be executed in
-## the scope of the call that rose the error. For example, the function
-## @code{print_usage()} when called from ## the Octave prompt rises this error.
+## Indicates the error was generated by an operation that cannot be executed in
+## the scope from which it was called.  For example, the function
+## @code{print_usage ()} when called from the Octave prompt raises this error.
 ##
 ## @item Octave:invalid-input-arg
 ## Indicates that a function was called with invalid input arguments.
 ##
 ## @item Octave:invalid-fun-call
-## Indicates that a function was called in an incorrect way, e.g. wrong number
+## Indicates that a function was called in an incorrect way, e.g., wrong number
 ## of input arguments.
 ##
 ## @item Octave:invalid-indexing
-## Indicates that a data-type was indexed incorrectly, e.g. real-value index for
-## arrays, inexistent field of a structure
+## Indicates that a data-type was indexed incorrectly, e.g., real-value index
+## for arrays, non-existent field of a structure.
 ##
 ## @end table
 
 
 function error_ids ()
   help ("error_ids");
 endfunction
 
+
 ## Remove from test statistics.  No real tests possible
 %!assert (1)
diff --git a/scripts/miscellaneous/module.mk b/scripts/miscellaneous/module.mk
--- a/scripts/miscellaneous/module.mk
+++ b/scripts/miscellaneous/module.mk
@@ -15,16 +15,17 @@ miscellaneous_FCN_FILES = \
   miscellaneous/computer.m \
   miscellaneous/copyfile.m \
   miscellaneous/debug.m \
   miscellaneous/delete.m \
   miscellaneous/dir.m \
   miscellaneous/dos.m \
   miscellaneous/dump_prefs.m \
   miscellaneous/edit.m \
+  miscellaneous/error_ids.m \
 	miscellaneous/fact.m \
   miscellaneous/fileattrib.m \
   miscellaneous/fileparts.m \
   miscellaneous/fullfile.m \
   miscellaneous/getappdata.m \
   miscellaneous/getfield.m \
   miscellaneous/gunzip.m \
   miscellaneous/gzip.m \
